### PR TITLE
handle commands endpoint python

### PR DESCRIPTION
**Pull request by ajwootto**
- add endpoint for handling commands issued to a location
- reorganize route handlers into their own files

### CODE REVIEW

**Comment by ajwootto** [comment]
Current dependencies on/for this PR:
* main
  * **PR #48** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/test-harness/48" target="_blank">Graphite</a> 
    * **PR #49** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/test-harness/49" target="_blank">Graphite</a> 
      * **PR #50** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/test-harness/50" target="_blank">Graphite</a> 
        * **PR #51** <a href="https://app.graphite.dev/github/pr/DevCycleHQ/test-harness/51" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/DevCycleHQ/test-harness/51?utm_source=stack-comment).

File: proxies/python/handlers/config.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from devcycle_python_sdk import Configuration, DVCClient
+
+def handle_config(body, data_store):
+    client_id, sdk_key, options = [body.get(k, None) for k in ('clientId', 'sdkKey', 'options')]
+
+    if (client_id == None):
+        error = {
+            "error": "Invalid request: missing clientId"
```
Comment (suthar26) [inline proxies/python/handlers/config.py:8]:
```suggestion
            "errorMessage": "Invalid request: missing clientId"
```

File: proxies/python/handlers/client.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        error = {
+            "errorMessage": "Invalid request: missing clientId"
+        }
+
+        return error, 400
+
+    configuration = Configuration()
+    configuration.api_key['Authorization'] = sdk_key
+
+    # TODO remove this when SDK properly supports passing in this property
```
Comment (ajwootto) [inline proxies/python/handlers/client.py:18]: did some cheating here to fix some of the incompatibilities at the proxy level just so I could get the tests to run at all

File: proxies/python/handlers/command.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    parsed_params = parse_params(params if params else [], data_store)
+
+    if not stored_entity:
+        return {
+            "errorMessage": "Invalid request: missing entity"
+        }, 404
+
+    if not command:
+        return {
+            "errorMessage": "Invalid request: misisng command"
```
Comment (suthar26) [inline proxies/python/handlers/command.py:51]:
```suggestion
            "errorMessage": "Invalid request: missing command"
```

File: proxies/python/handlers/command.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+    data_store['commandResults'][command][command_id] = result
+
+    result_data = result.to_dict() if hasattr(result, "to_dict") else result
+
+    return {
+        "entityType": entity_type,
+        "data": camel_case_dict(result_data),
+        "logs": []
+    }, 200, {
```
Comment (suthar26) [inline proxies/python/handlers/command.py:71]: 200 vs 201, since technically it is an object created

Comment (ajwootto) [reply proxies/python/handlers/command.py:71]: I think 201 makes sense here but the Nodejs proxy is currently responding 200 on this endpoint

Comment (suthar26) [reply proxies/python/handlers/command.py:71]: will change that.

File: proxies/python/handlers/command.py [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        return data_store["users"].get(user_id, None)
+
+    return None
+
+def parse_params(params, data_store):
+    parsed_params = []
+    for element in params:
+        if element.get("location", None):
+            entity = get_entity_from_location(element["location"], data_store)
+            parsed_params.append(entity)
```
[the remark this answers is not in the document]
Comment (suthar26) [reply proxies/python/handlers/command.py:26]: will create a ticket for it

File: proxies/python/helpers/camelcase.py [hunk 6]
```diff
@@ -0,0 +1,10 @@
+from re import sub
+
+
+def camel_case(s):
+    s = sub(r"(_|-)+", " ", s).title().replace(" ", "")
+    return ''.join([s[0].lower(), s[1:]])
+
+
+def camel_case_dict(source):
+    return {camel_case(key): val for key, val in source.items()}
```
Comment (suthar26) [inline proxies/python/helpers/camelcase.py:10]: add new line character after this 
```suggestion
    return {camel_case(key): val for key, val in source.items()}
    
```

**Review by suthar26 — APPROVED** [review]

File: proxies/python/handlers/command.py [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+def handle_command(path, body, data_store):
+    entity, command, params, is_async = [body.get(k, None) for k in ('entity', 'command', 'params', 'isAsync')]
+
+    stored_entity = get_entity_from_location(path, data_store)
+    parsed_params = parse_params(params if params else [], data_store)
+
+    if not stored_entity:
+        return {
+            "errorMessage": "Invalid request: missing entity"
```
Comment (suthar26) [inline proxies/python/handlers/command.py:46]: I thought we were changing these from `errorMessage` to `message` to match other APIs 
```suggestion
            "message": "Invalid request: missing entity"
```

File: proxies/python/handlers/command.py [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+        }, 404
+
+    if None in parsed_params:
+        return {
+            "errorMessage": "Invalid request: missing entity from param"
```
Comment (suthar26) [inline proxies/python/handlers/command.py:51]: I thought we were changing these from `errorMessage` to `message` to match other APIs 

```suggestion
            "message": "Invalid request: missing entity from param"
```

File: proxies/python/handlers/command.py [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+        }, 404
+
+    if not command:
+        return {
+            "errorMessage": "Invalid request: missing command"
```
Comment (suthar26) [inline proxies/python/handlers/command.py:56]: I thought we were changing these from `errorMessage` to `message` to match other APIs 
```suggestion
            "message": "Invalid request: missing command"
```

**Review by suthar26 — CHANGES_REQUESTED** [review]

**Review by suthar26 — APPROVED** [review]

**Pull request merged**